indico/htdocs/js/indico/jquery/palettepicker.js

```diff
@@ -111,7 +112,12 @@
                 events: {
                     show: self._updateSelection.bind(self)
                 }
-            });
+            }
+            if (self.options.qtipConstructor) {
+                self.options.qtipConstructor(element, qtipOptions);
```

Comment:
My personal preference is to have an explicit argument but if someone else think using `this` is clearer I'll change it.